data/json/mapgen/military/mil_base/mil_base_z0.json

```diff
@@ -675,9 +675,9 @@
         "V": [ { "item": "tools_common", "chance": 50, "repeat": [ 0, 1 ] } ],
         "U": [
           { "item": "SUS_office_filing_cabinet", "chance": 50, "repeat": [ 1, 10 ] },
-          { "item": "guns_obscure", "chance": 50, "repeat": [ 1, 2 ] },
```

Comment:
This was a recurring pattern, the obscure guns groups were getting abused in a number of places, and they have really wacky guns in them.